packages/firestore/rollup.config.js

```diff
@@ -96,15 +88,37 @@ const es5Builds = [
       )
   },
   /**
-   * Browser Builds
+   * Browser CJS Build
+   * 
+   * The Browser CJS build is not mangled as Terser's property name mangling 
```

Comment:
Is CJS format the root of the issue? I thought only the Node build is affected.

Reply:
The CJS build that we published contains a bunch of variables prefixed with __PRIVATE_, which are not renamed by Terser. As such, the file is bigger than before.
`firebase-firestore.js` uses the ESM build, before converting to CJS, which means it is not affected.